HaulageApplication/HaulageAppTests/ItemTests.cs

```diff
@@ -0,0 +1,100 @@
+using HaulageApp.Data;
+using HaulageApp.Models;
+using HaulageApp.ViewModels;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using Moq;
+
+namespace HaulageAppTests;
+
+public class ItemTests
+{
+    private readonly Mock<DbSet<Item>> _mockItems;
+    private readonly Mock<HaulageDbContext> _mockContext;
+    private readonly Mock<ILogger<ItemViewModel>> _loggerMock;
+    private readonly ItemViewModel _viewModel;
+    private readonly Bill _bill;
+    private readonly Trip _trip;
+    private readonly Item _item;
+
+    public ItemTests()
+    {
+        _mockContext = new Mock<HaulageDbContext>();
+        _mockItems = new Mock<DbSet<Item>>();
+        _loggerMock = new Mock<ILogger<ItemViewModel>>();
+        _viewModel = new ItemViewModel(_mockContext.Object, _loggerMock.Object);
+
+        _bill = new Bill
+        {
+            BillId = 1, CustomerId = 1, Amount = 100.20m, Status = "pending", Items = new List<Item>
+            {
+                new Item { ItemId = 1, PickupLocation = "Loc A", DeliveryLocation = "Loc B", Status = "delivered" },
+                new Item { ItemId = 2, PickupLocation = "Loc C", DeliveryLocation = "Loc D", Status = "pending" }
+            }
+        };
+
+        _trip = new Trip { Id = 1 };
+
+        _item = new Item
+        {
+            ItemId = 3, PickupLocation = "Loc C", DeliveryLocation = "Loc D", Status = "pending"
+        };
+
+        _mockContext.Setup(e => e.item).Returns(_mockItems.Object);
+    }
+
+    [Fact]
+    public void ShouldAssignItemWhenQureied()
```

Comment:
Different naming convention. I would name this like this to make it consistent with others: 
- ApplyQueryAttributes_ShouldAssignItemProperties

Reply:
Will update